component/src/main/java/io/siddhi/extension/io/cdc/source/polling/CDCPoller.java

```diff
@@ -185,6 +190,9 @@ public void resume() {
     public void run() {
         try {
             pollingStrategy.poll();
```

Comment:
Shouldn't this come in the else part? In this scenario, whenever the cdc source starts running, a poll() happens right. Please confirm

Reply:
`poll()` is called to set the lastReadPollingColumnValue. If the `poll()` is called in the `scheduleJob()` then records which has been inserted between the time app starts running and the first cronschedule will be missed. To ignore this issue the lastReadPollingColumnValue has to be stored. So it shouldn't come in the else part.